test/system/src/main/java/io/pravega/test/system/framework/services/marathon/PravegaControllerService.java

```diff
@@ -122,18 +121,18 @@ private App createPravegaControllerApp() {
 
         String componentCode = "controller";
 
-        //set env
+        // These properties are defined in "io.pravega.controller.util.Config".
```

Comment:
This is not consistent with the PR description. 

In the description you say "remove Controller dependencies".

Here, you are removing references **from the System test package** to the Controller.

A few issues that I'd like resolved:
1. Make the PR description consistent with the contents of the PR.
2. Remove the dependency on the controller project from this project's `build.gradle` section.
3. I am concerned that this change in particular (swapping `Config.*` with hardcoded strings) will break if we make a change to those properties at one point.

Reply:
In response to #3. Certainly a possibility. I took inspiration from `PravegaSegmentStoreService.java` which also uses hard-coded values to set config options. Can certainly refactor the controller's `Config` class into a shared package if that's a better approach.